servers/movie_writer/movie_writer.cpp

```diff
@@ -33,6 +33,7 @@
 #include "core/io/dir_access.h"
 #include "core/os/time.h"
 #include "servers/display_server.h"
+#include "servers/rendering/rendering_server_default.h"
```

Comment:
Does this need RenderingServerDefault, instead of just RenderingServer?
```suggestion
#include "servers/rendering_server.h"
```